gtsfm/evaluation/metrics_report.py

```diff
@@ -39,13 +41,42 @@ def get_readable_metric_name(metric_name: str) -> str:
 def create_table_for_scalar_metrics(metrics_dict: Dict[str, Union[float, int]]) -> str:
     """Creates a table in HTML format for scalar metrics.
 
+    Args:
+        metrics_dict: A dict, where keys are names of metrics and values are
+        the dictionary representation of the metric.
+
     Returns:
         Table with scalar metrics and their values in HTML format.
     """
     table = {"Metric name": list(metrics_dict.keys()), "Value": list(metrics_dict.values())}
     return tabulate(table, headers="keys", tablefmt="html")
 
 
+def create_table_for_scalar_metrics_and_compare(metrics_dict: Dict[str, List[Union[float, int]]]) -> str:
+    """Creates a table in HTML format for scalar metrics from multiple SfM pipelines (GTSfM, COLMAP).
+
+    Args:
+        metrics_dict: A dict, where keys are names of metrics and values are
+        the dictionary representation of the metric.
```

Comment:
values are not Dict in the type hint for the argument...

Reply:
Updated the comment.